src/util/visit_helper.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+#include "sat_literal.h"
```

Comment:
not a fan of adding things to util directory that depend on sat::literal.
This is done in a different instance, but would like to avoid it.
Can move to sat_types.h